zypper-docker.go

```diff
@@ -22,8 +22,12 @@ import (
 )
 
 var exitWithCode func(code int)
+var killChannel chan bool
 
 func main() {
+	killChannel = make(chan bool)
```

Comment:
Maybe killChannel can be initialized inside the `listenSignals` function? If you do that, maybe you should move the global declaration of killChannel to the signals.go file